packages/fxa-graphql-api/pm2.config.js

```diff
@@ -21,6 +21,9 @@ module.exports = {
         TS_NODE_FILES: 'true',
         PORT: '8290', // TODO: this needs to get added to src/config.ts
         CUSTOMS_SERVER_URL: 'none',
```

Comment:
did you mean to update this to the `http://localhost:7000` default, or does this override that default anyway?

Reply:
This overrides the default so I removed default since it wasn't being used